internal/instance/resource_instance.go

```diff
@@ -116,6 +117,21 @@ func (r InstanceResource) Schema(_ context.Context, _ resource.SchemaRequest, re
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.RequiresReplace(),
 				},
+        Validators: []validator.String{
```

Comment:
Please check the formatting with `gofmt`. In Go indentation is done with `tab` (and not spaces).